swap/src/cli/command.rs

```diff
@@ -321,6 +332,30 @@ enum CliCommand {
     },
     /// Show a list of past, ongoing and completed swaps
     History,
+    /// Output all logging messages that have been issued.
+    Logs {
+        #[structopt(
+            short = "d",
+            help = "Print the logs from this directory instead of the default one."
+        )]
+        logs_dir: Option<PathBuf>,
+        #[structopt(
```

Comment:
I think we could remove this for simplicity sake but I'll let you decide. I'd also be okay with leaving it in.

Reply:
(the write to file argument)